source/jormungandr/tests/check_utils.py

```diff
@@ -624,6 +624,9 @@ def is_valid_section(section, query):
 
     assert section['type']  # type cannot be empty
 
+    if section['type'] == 'public_transport':#, 'on_demand_transport':
+        is_valid_rt_level(section['data_freshness'])
```

Comment:
we may also add a consistency check (like `base_arrival != arrival => freshness != base`)